release_notes.py

```diff
@@ -43,7 +44,7 @@
 
 
 def add_dot(text):
-    if not text:
+    if not text or len(text) < 2:
```

Comment:
why?
dash release notes should not get to this point.

Reply:
It failed on some release notes on the next line somehow, maybe because of not enforcing changlog on some of 5.5 changes. it'll prevent the script from failing